samples/colab/README.md

```diff
@@ -1,49 +1,19 @@
-
-Compile, train and execute a TensorFlow Keras neural network with IREE
-
-[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openxla/iree/blob/main/samples/colab/mnist_training.ipynb)
+These [Colab](https://colab.google/) notebooks contain interactive sample
+applications using IREE's Python bindings and ML framework integrations.
 
-### [resnet\.ipynb](resnet.ipynb)
-
-Loads a pretrained
-[ResNet50](https://www.tensorflow.org/api_docs/python/tf/keras/applications/ResNet50)
-model and runs it using IREE
-
-[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openxla/iree/blob/main/samples/colab/resnet.ipynb)
-
-### [tensorflow_hub_import\.ipynb](tensorflow_hub_import.ipynb)
-
-Downloads a pretrained
-[MobileNet V2](https://tfhub.dev/google/tf2-preview/mobilenet_v2/classification)
-model, pre-processes it for import, then compiles it using IREE
-
-[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openxla/iree/blob/main/samples/colab/tensorflow_hub_import.ipynb)
-
-### [tflite_text_classification\.ipynb](tflite_text_classification.ipynb)
-
-Downloads a pretrained
-[TFLite text classification](https://www.tensorflow.org/lite/examples/text_classification/overview)
-model, and runs it using TFLite and IREE
+## Notebooks
 
-[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openxla/iree/blob/main/samples/colab/tflite_text_classification.ipynb)
+Framework | Notebook file | Description | Link
+--------  | ------------- | ----------- | ----
+Generic | [low_level_invoke_function\.ipynb](low_level_invoke_function.ipynb) | Shows off some concepts of the low level IREE python bindings | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openxla/iree/blob/main/samples/colab/low_level_invoke_function.ipynb)
+PyTorch | [pytorch_jit\.ipynb](pytorch_jit.ipynb) | Uses [SHARK-Turbine](https://github.com/nod-ai/SHARK-Turbine) for eager execution in a PyTorch session | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openxla/iree/blob/main/samples/colab/pytorch_jit.ipynb)
+TensorFlow | [edge_detection\.ipynb](edge_detection.ipynb) |Performs image edge detection using TF and IREE | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openxla/iree/blob/main/samples/colab/edge_detection.ipynb)
+TensorFlow | [mnist_training\.ipynb](mnist_training.ipynb) | Compile, train, and execute a neural network with IREE | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openxla/iree/blob/main/samples/colab/mnist_training.ipynb)
```

Comment:
May rename these notebooks to `tensorflow_` or `tf_` to make it easier to see the alphabetical ordering and grouping.

Reply:
Will leave that to a follow-up, since more files will need to be updated as part of renaming.

Reply:
Or remove them. In practice we don't have a maintainer for tensorflow and it hurts every time we have to bake an excuse to people.

Reply:
Rename is in https://github.com/openxla/iree/pull/15155

Yeah... I feel the maintenance and support pain... maybe I'll remove or simplify the existing notebooks after standing up new ones in PyTorch (and ideally also JAX).